getData.py

```diff
@@ -0,0 +1,131 @@
+# https://fmpcloud.io/api/v3/income-statement/AAPL?period=quarter&apikey=18bb80696ecc77ff0ebaf2056f075aad
+# https://fmpcloud.io/api/v3/historical-price-full/AAPL?serietype=line&apikey=18bb80696ecc77ff0ebaf2056f075aad
+import json
+import requests
+import os
+import argparse
+
+api_key = "18bb80696ecc77ff0ebaf2056f075aad"
+prefix_url = "https://fmpcloud.io/api/v3"
+
+
+def str2bool(v):
+    if isinstance(v, bool):
+        return v
+    if v.lower() in ('yes', 'true', 't', 'y', '1'):
+        return True
+    elif v.lower() in ('no', 'false', 'f', 'n', '0'):
+        return False
+    else:
+        raise argparse.ArgumentTypeError('Boolean value expected.')
+
+
+def get_urls(symbol):
+    income_url = prefix_url + '/income-statement/' + symbol + '?period=quarter'
+    income_url += '&apikey=' + api_key
+    historic_url = prefix_url + '/historical-price-full/' + symbol + '?period=quarter'
+    historic_url += '&apikey=' + api_key
+    return [income_url, historic_url]
+
+
+def get_results(symbol, urls):
+    get_url_data(symbol, 'income', urls[0])
+    get_url_data(symbol, 'historical', urls[1])
+
+
+def get_url_data(symbol, name, url):
+    filename = os.path.join('raw', symbol + '_' + name + '.json')
+    print(filename)
+    with open(filename, 'w') as file:
+        response = requests.get(url)
+        if response.status_code == 200:
+            data = response.content.decode('utf-8')
+            file.write(data)
+        else:
+            print("failed to get " + name)
```

Comment:
might be a good change to use the standard `logging` module (it has lots of documentation but only two lines to make it work..)

```
import logging
logging.basicConfig(...)
...
logging.error(...)
```